src/components/model/ModelForm.svelte

```diff
@@ -200,11 +204,15 @@
     </Input>
     <Input layout="inline">
       <label for="view">Default View</label>
-      <select name="view" class="st-select w-100" bind:value={viewId}>
-        <option value={null}>None</option>
-        {#each views as viewOption}
-          <option value={viewOption.id}>{viewOption.name} (ID: {viewOption.id})</option>
-        {/each}
+      <select name="view" class="st-select w-100" bind:value={viewId} disabled={viewsLoading}>
+        {#if viewsLoading || !views}
```

Comment:
Can we just rely on `viewLoading` here or do we also have to still check if `views` is defined?

Reply:
Ohh makes sense. Thanks!